dev-libs/sway/sway-1.0_rc2.ebuild

```diff
@@ -65,36 +72,35 @@ src_prepare() {
 
 src_configure() {
 	local emesonargs=(
-		$(meson_use bash-completion bash-completions)
-		$(meson_use fish-completion fish-completions)
+		-Dman-pages=$(usex man enabled disabled)
+		-Dtray=$(usex tray enabled disabled)
+		-Dxwayland=$(usex X enabled disabled)
 		$(meson_use wallpapers default-wallpaper)
+		$(meson_use fish-completion fish-completions)
 		$(meson_use zsh-completion zsh-completions)
-		"-Dtray=$(usex tray enabled disabled)"
-		"-Dxwayland=$(usex X enabled disabled)"
+		"-Dbash-completions=true"
 		"-Dwerror=false"
 	)
+
 	if use swaybar || use swaybg; then
 		emesonargs+=("-Dgdk-pixbuf=enabled")
 	else
 		emesonargs+=("-Dgdk-pixbuf=disabled")
 	fi
-	if [[ ${PV} != 9999 ]]; then
-		emesonargs+=("-Dsway-version=${PV}")
-	fi
```

Comment:
it looks like sway-version was removed from rc2 (and 9999)

Reply:
You commented an outdate code. I've already removed it.